### ISSUE TITLE
#11 has some problem.

### ISSUE THREAD

**Issue by yeseuleee** [golems/ach#12]
I have installed ach (version before commit 592fe29 ).
When I was install it, I don't experienced any problem.
But when I have installed ach today, I experienced some problem.

The error is

**Comment by ndantam** [comment]
Could you please give some additional detail to help reproduce the problem:
* Does the problem occur on a clean clone?
* What OS / distribution are you using?
* Which build step is failing (autoreconf, configure, make)?

**Comment by yeseuleee** [comment]
Sorry. I was add more detail.
Does the problem occur on a clean clone?
-> Yes. The problem occur on a clean clone that recently version.
What OS / distribution are you using?
-> My OS is Ubuntu 16.04 LTS
Which build step is failing (autoreconf, configure, make)?
-> Failing in "make" build step 

Really Thankyou!

**Comment by ndantam** [comment]
Thanks for the details.  I think the best approach is to set up dockerized integration tests (as we've done for OMPL, TMKit, and Amino) to reproduce this issue and catch similar problems in the future.  Unfortunately, I'm traveling the for the next week and have some tight deadlines soon, so I'm not sure if I can get to this right way.  But, it sounds like you are OK for now using the previous commit or the last release version?

**Comment by yeseuleee** [comment]
Yes. The problem wasn't occur in previous commit. So I use previous revision. Just I wanted let you know

**Comment by acxz** [comment]
@yeseuleee those commits were needed due to upgrades to autoconf and man2html.
Can you list the versions you are currently using for these packages? i.e. what is the output of `autoconf --version` and the last section of the output of `man2html --help`? Maybe the fix might just be a simple `sudo apt update && sudo apt install autoconf man2html` on Ubuntu 16.04.

**Comment by bsoe-flexiv** [comment]
I receive the same error. This is the only output from running make:
Makefile:2525: *** missing separator.  Stop.

Line 2525:
@DX_RULES@

Checking out commit 4a38f91 compiles fine.

System:
Ubuntu 18.04
automake 1.15.1
autoconf 2.69
man2html 1.6g-11)

**Comment by crackwitz** [comment]
I've seen that error too, but only on my advisor's computer, which is an old ubuntu.

it seems to be related to doxygen. see if installing doxygen makes a difference.

**Comment by bsoe-flexiv** [comment]
I have doxygen 1.8.13 installed. I think the issue is related to autoconf-archive. I do not have autoconf-archive installed because it broke a different part of the build process (sorry, I dont recall what).
